src/core/agent/records/identifierMetadataRecord.ts

```diff
@@ -30,6 +31,7 @@ class IdentifierMetadataRecord extends BaseRecord {
   theme!: number;
   multisigManageAid?: string | undefined;
   groupMetadata?: groupMetadata;
+  authorizedEids?: string[] | undefined;
```

Comment:
can remove `| undefined` - it doesn't matter here but I've seen people before doing `?: string[] | null` which allows both null and undefined. Safer to just use `?`.